monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -48,7 +48,7 @@ link:#thanosrulerconfig[ThanosRulerConfig]
 
 [options="header"]
 |===
-| Property | Type | Description 
+| Property | Type | Description
 |apiVersion|string|Defines the API version of Alertmanager. Possible values are `v1` or `v2`. The default is `v2`.
```

Comment:
Could this be the case?
```suggestion
|apiVersion|string|Defines the API version of Thanos Ruler. Possible values are `v1` or `v2`. The default is `v2`.
```

Reply:
No. The issue that this PR addresses was that Alertmanager was mistakenly listed as a component in the thanosRulerConfig settings.

Reply:
Thank you for the explanation.